source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -88,8 +88,37 @@ title: Verify the data encryption key was created
 ref: verify-data-key-created
 level: 4
 content: |
-  Verify that the key document was created in your MongoDB replica set in the
-  key vault collection. This document contains the following data:
+  First, verify that the command to create the data encryption key returns the
+  ID of the newly-created key:
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        The ``createDataKey()`` method returns a :java-docs-latest:`BsonBinary
```

Comment:
It seems like we should mention `createDataKey()` in the previous step (**B.3 Create a Data Encryption Key**) but we don't currently.

- We should add `createDataKey()` to the end of the previous step's code block
- Seems like this new block would make sense as part of the previous step as well. Thoughts?

Reply:
Makes sense to add the call `createDataKey`, looks like we just missed that part. I think the "verify" step could stay separate since it already has 3 sub-sections: print the returned key, what the output should look like, and what the key vault collection should now contain.
We should probably show the user the commands to check the `__keyVault` collection as well. What do you think?

Reply:
👍 Fine with separate if you think it works!

RE: the keyvault commands, would that just be a `mongoClient.db("encryption").collection("__keyVault").find()` or are you thinking of something else? I don't think we _need_ to have it but if you think of a nice way to slot it in go for it.